### DISCUSSION TITLE
Anyone paying attention?

### DISCUSSION THREAD

**Discussion by MarkLindsay** [jasonrohrer/AnotherPlanetData#16] [Polls]
If you read this you are legally obliged to vote.

**Comment by MarkLindsay** [comment]
K. Not a lot of traffic.

**Reply by connorhsm** [reply]
What are you looking for? There's an unofficial discord that is rather lively. https://discord.gg/Bgnt7eXz

**Reply by MarkLindsay** [reply]
Thanks. I’m aware. Just curious if anyone is poking around like I am.
